### PR TITLE
Problem: Travis does not install docs build-deps

### PR DESCRIPTION
**Pull request by bluca**
Solution: add build-dependencies and regenerate from zproject to test and deploy in releases all the documentation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,14 +20,18 @@ save.xml
 
 # Executables
 src/czmq_selftest
-src/makecert
```
Comment (jimklimov) [inline .gitignore]: Strange... did these deliverables disappear, or are they not listed in project.xml?

File: .gitignore [hunk 2]
```diff
@@ -84,12 +99,13 @@ src/app/jni/output
 # Python build directory
 build/
 
-# MSVC build directories
```
Comment (jimklimov) [inline .gitignore]: I think this block also has to stay, for portable projects at least. Probably `zproject_git.gsl` should take care of this (not a CZMQ issue except further regeneration of the project when that fix lands).

File: .gitignore [hunk 3]
```diff
@@ -84,12 +99,13 @@ src/app/jni/output
...
+# Qt
+*pro.user
+moc_*
+build-*
+bindings/qt/selftest/selftest
 
 # Temporary files
 *.swp
+*.bak
 *~
```
Comment (jimklimov) [inline .gitignore]: Would it make sense to keep this in common `zproject_git.gsl` as well?

File: .gitignore (hunk 1 above)
Comment (sappo) [inline .gitignore]: See https://github.com/zeromq/czmq/commit/d6ffdcc11e7feb997302e4982a95bf140ba058a2

File: .gitignore (hunk 3 above)
Comment (sappo) [inline .gitignore]: In general yes, but there are always a couple of exceptions which differ for each project.

File: .gitignore (hunk 2 above)
Comment (bluca) [inline .gitignore]: makes sense

File: .gitignore (hunk 1 above)
Comment (jimklimov) [inline .gitignore]: Thanks for the link. Seems ok :)

**Pull request merged**